Feat - add eslint

floragc-octo opened the pull request.

NerOcrO commented on package.json:
```diff
@@ -1,26 +1,30 @@
+  "version": "0.0.1",
+  "description": "Browsers' extension to highlight dev environment",
+  "scripts": {
+    "package": " zip -r extension.zip ./extension/*",
```
```suggestion
    "package": "zip -r extension.zip ./extension/*",
```

NerOcrO commented on .eslintrc.json:
```diff
@@ -0,0 +1,15 @@
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
```
```suggestion

```

NerOcrO commented:
Il ne peut pas avoir `package-lock.json` et `yarn.lock` en même temps, choisie ton camp !

sibe-octo committed:
Feat - add eslint

tech remove package.lock and create gitignore file

tech update package.lock

tech add lint command and fix linter rules

bsr autofix lint

tech fix lint on files and remove dead code

tech add linter

sibe-octo committed:
tech fix lint
